src/stories/components/Forms/Select/styles.ts

```diff
@@ -0,0 +1,32 @@
+import MuiFormControl from '@mui/material/FormControl';
+import MuiFormHelperText from '@mui/material/FormHelperText';
+import MuiInputLabel from '@mui/material/InputLabel';
+import MuiMenuItem from '@mui/material/MenuItem';
+import MuiSelect from '@mui/material/Select';
+import { styled } from '@mui/material/styles';
+
+import { Option as TOption } from './types/Option';
+
+export const Wrapper = styled(MuiFormControl)`
+  width: 100%;
+`;
+
+export const Label = styled(MuiInputLabel)``;
+
+Label.defaultProps = {
```

Comment:
Sugiro colocar as `defaultProps` for do arquivo de estilização, pode colocar diretamente no componente